ibm_cloud_sdk_core/utils.py

```diff
@@ -16,8 +16,8 @@
 # from ibm_cloud_sdk_core.authenticators import Authenticator
 import datetime
 import json as json_import
-from os import getenv, environ
-from os.path import dirname, isfile, join, expanduser, abspath
+from os import getenv, environ, getcwd
+from os.path import isfile, join, expanduser, abspath
```

Comment:
```suggestion
from os.path import isfile, join, expanduser
```

This change will fix the linter error that is causing the build to fail.

Reply:
I removed the unused path and I changed a test to load the credentials file from the current working directory instead of the absolute path of ../../__file__ but that doesn't test the exact reason this error happened. It happened because the core was being used one directly higher in the python sdk. This is fixed but I currently dont know an easy was of testing that scenario.

Reply:
Can you have a test that relies on a file in a specific directory X  (e.g. X/ibm-credentials.env) where the test first performs the equivalent of the "cd X" shell command? Then when you trigger the loading of the credentials file, the one in the (now) "current" directory should be found.

Reply:
Good idea, I now modified two tests. For test_iam, I changed it to load the ibm-credentials.env file from the current working directory instead of from the file_path environment variable. For test_configure_services, I did what your comment suggested and changed working directories to the credentials file, then triggered loading the credentials file in the new current directory.